prov/rxd/src/rxd.h

```diff
@@ -246,6 +250,8 @@ struct rxd_pkt_entry {
 	struct rxd_pkt *pkt;
 };
 
+#define rxd_div_ceil(a, b) ((a / b) + (a % b ? 1 : 0))
```

Comment:
I think this can be
```
((a + b - 1) / b)
```
which avoids a branch and a double division.  This macro can be moved into common code